tests/phpunit/tests/query/php8xCompat.php

```diff
@@ -0,0 +1,80 @@
+<?php
```

Comment:
[[53891]](https://core.trac.wordpress.org/changeset/53891) introduced tests in `tests/phpunit/tests/query/parseQuery.php` - would it be worth following in the same place for consistency?